python/tvm/meta_schedule/cost_model/xgb_model.py

```diff
@@ -35,7 +35,15 @@
 from ..utils import cpu_count, derived_object, shash2hex
 from .metric import max_curve
 
+
 if TYPE_CHECKING:
+    try:
```

Comment:
if we put the import here builds in CI could fail. I think this is due to fact that we need this import instead of using `xgb` only as a type hint. @zxybazh Should we put it out of the `TYPE_CHECKING` instead?